ads/google/imaVideo.js

```diff
@@ -585,17 +595,23 @@ function changeIcon(element, name, fill = '#FFFFFF') {
  * @param {!Object} global
  * @visibleForTesting
  */
-export function onClick(global) {
-  playbackStarted = true;
-  uiTicker = setInterval(uiTickerClick, 500);
-  setInterval(playerDataTick, 1000);
-  bigPlayDiv.removeEventListener(interactEvent, onClick);
```

Comment:
One question: do we need to `removeEventListener` to interactEvent, and add it back when allAdsCompleted or onContentResume? Personally I don't know how much resource can it save give  we've already set display to none. Up to you.

Reply:
Ahhh so we don't want to remove the event listener here, because we may need it in the future. Since the big play div is hidden, the event listener wont be fired, and unlike the controls, it doesn't conflict with another event listener on the video itself 😄

Reply:
Discussed offline. Will remove/add as needed

Reply:
Discussed more offline, this falls more under a seperate refactoring PR. Since previously there was no method of adding/removing dynamically and all that.